### DISCUSSION TITLE
Welcome to tenant-buckets Discussions!

### DISCUSSION THREAD

**Discussion by secrethash** [fourdotsix/tenant-buckets#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vizcainomc66** [comment]
Hi, Why did the package stop? I'm trying to do this with tenancyforlaravel and vapor without success. I try to implement this package but doesn't work with real s3. Did exist a new solution?

**Reply by secrethash** [reply]
Actually I think it doesn't work properly with Vapor for some reason. Refer to #8
